velox/exec/tests/PrintPlanWithStatsTest.cpp

```diff
@@ -265,8 +265,8 @@ TEST_F(PrintPlanWithStatsTest, partialAggregateWithTableScan) {
          {"   Output: .+, Cpu time: .+, Blocked wall time: .+, Peak memory: .+, Memory allocations: .+, Threads: 1"},
          {"      dataSourceLazyWallNanos\\s+sum: .+, count: 1, min: .+, max: .+"},
          {"      distinctKey0\\s+sum: .+, count: 1, min: .+, max: .+"},
-         {"      hashtable.capacity\\s+sum: 1252, count: 1, min: 1252, max: 1252"},
-         {"      hashtable.numDistinct\\s+sum: 835, count: 1, min: 835, max: 835"},
+         {"      hashtable.capacity\\s+sum: 1273, count: 1, min: 1273, max: 1273"},
```

Comment:
Why this is related to this change or is a flakiness on head? Thanks!

Reply:
This would be caused by flakiness as the test relies on random data generation. I saw it locally so pushed to check if it resolves the test failures.

Reply:
It seems these changes are not needed. Reverting them and re-running tests.